dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/apps/kafka/MyDataDeserializer.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.reactive.messaging.fat.apps.kafka;
+
+import org.apache.kafka.common.serialization.Deserializer;
+
+/**
+ *
+ */
+public class MyDataDeserializer implements Deserializer<MyData> {
+
+    /** {@inheritDoc} */
+    @Override
+    public MyData deserialize(String topic, byte[] data) {
+        String dataStr = new String(data);
```

Comment:
I think you'll need to specify the character set here, otherwise the test may fail in cases where the platform encoding isn't compatible with UTF-8.

The Kafka String (de)serializer defaults to UTF-8 unless configured otherwise.

Reply:
done